packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1374,22 +1373,57 @@ class FocusManager with DiagnosticableTreeMixin {
   // The set of nodes that need to notify their listeners of changes at the next
   // update.
   final Set<FocusNode> _dirtyNodes = <FocusNode>{};
+  FocusNode _unfocusedNode;
```

Comment:
I don't fully understand what the purpose of this property is.

Reply:
I added a comment, but it's the node that is marked to be unfocused the next time the focus changes are applied.

Reply:
I see. I think this has an unfortunate name then. From the name I got that this node has already been unfocused. Instead it's more future looking: This is the node that will get unfocused. I wonder if we can get a better name. (The comment already helps).

Reply:
OK, I renamed this to `_markedForUnfocus`, and I also renamed `_nextFocus` to `_markedForFocus`.